models/unittest/testdb.go

```diff
@@ -89,6 +89,7 @@ func MainTest(m *testing.M, testOpts *TestOptions) {
 	}
 
 	if err = CreateTestEngine(opts); err != nil {
+		_, _ = fmt.Fprintln(os.Stderr, `sqlite3 requires: import _ "github.com/mattn/go-sqlite3" or -tags sqlite,sqlite_unlock_notify`)
```

Comment:
? Are you sure the returned error is caused by non imported sqlite3?

Reply:
Yes. Even if not, it doesn't harm any one.

Reply:
Maybe we should move these lines into the `CreateTestEngine`?

Reply:
done in b1778a051c0fd1cc58c20af1796e49e8bb2cb584